rubato/struct/scene.py

```diff
@@ -22,27 +23,39 @@ class Scene:
         ui (Group): The ui elements of this scene. These are drawn on top of everything else and do not interact with
             the other game objects.
         camera (Camera): The camera of this scene.
-        id (str): The id of this scene.
         border_color: The color of the border of the window.
         background_color: The color of the background of the window.
     """
 
     def __init__(
         self,
-        name: str = "default",
+        name: str | None = None,
         background_color: Color = Color(255, 255, 255),
         border_color: Color = Color(),
     ):
         self.root: Group = Group(name="root")
         self.ui: Group = Group(name="ui")
         self.camera = Camera()
         self._ui_cam = Camera()
-        self.id: str = name
+        self._id: str | None = name
         self.started = False
         self.border_color = border_color
         self.background_color = background_color
 
-        Game.scenes.add(self, name)
+        Game._add(self)
+
+    @property
+    def name(self) -> str:
+        """
+        The name of this scene. Get-only.
```

Comment:
```suggestion
        The name of this scene. Read-only.
```

I'm just nitpicking here. This is optional.